web/models.py

```
@@ -2098,57 +2107,53 @@
-
-        return reverse("quiz_detail", kwargs={"pk": self.pk})
-
-    @property
-    def question_count(self):
-        return self.questions.count()
-
-    @property
-    def completion_count(self):
-        return self.user_quizzes.filter(completed=True).count()
+    class Meta:
+        ordering = ["created_at"]
```

Ruff (0.8.2)

Order of model's inner classes, methods, and fields does not follow the Django Style Guide: `Meta` class should come before `__str__` method

(DJ012)